tests/main/lxd-no-fuse/task.yaml

```diff
@@ -1,4 +1,4 @@
-summary: Check that we give a useful error when fuse is missing in lxd
+summary: Check that we can install snaps when fuse is missing in lxd
```

Comment:
It's not missing though but installed automatically now due to dependencies.

Reply:
@stolowski so, fuse now is included into the core snap (commit 6d37325511b5531591f6d35f921cd62106281688) as a dependency, so even when we remove the fuse package the snaps can be installed. I also proposed to remove this test, because, as you said, now the test lacks sense but we decided to keep it just in case.

Reply:
I see. Can you add a comment or add description mentioning that fuse now comes with core?